app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_comment_counter
+  def update_post_comment_counter
+    post.comments_counter = 0 unless post.comments_counter?
+    post.comments_counter += 1
+    post.save
+  end
```

Comment:
[OPTIONAL] You've done a great job setting up your models, congrats! You could check out the `.increment` method, this will save you some lines of code, e.g: `post.increment!(:comments_counter)`